src/interpreter/plugin/SimpleArithmertic.ts

```diff
@@ -0,0 +1,217 @@
+      ],
+    },
+    'HF.UPLUS': {
+      method: 'uplus',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+  }
+
+  public add(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.ADD'),
+      (a: number, b: number) => this.interpreter.arithmeticHelper.addWithEpsilon(a, b)
+      )
+  }
+
+  public concat(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.CONCAT'),
+      (a: string, b: string) => a.concat(b)
+    )
+  }
+
+  public divide(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.DIVIDE'),
+      divide
+    )
+  }
+
+  public eq(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.EQ'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) === 0
+      )
+    )
+  }
+
+  public gt(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.GT'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) > 0
+      )
+    )
+  }
+
+  public gte(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.GTE'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) >= 0
+      )
+    )
+  }
+
+  public lt(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.LT'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) < 0
+      )
+    )
+  }
+
+  public lte(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.LTE'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) <= 0
+      )
+    )
+  }
+
+  public minus(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.MINUS'),
+      (a: number, b: number) => this.interpreter.arithmeticHelper.subtract(a, b)
+    )
+  }
+
+  public multiply(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.MULTIPLY'),
+      (a: number, b: number) => a*b
```

Comment:
Can we refactor this fragment and TIMES_OP https://github.com/handsontable/hyperformula/blob/master/src/interpreter/Interpreter.ts#L146 to a common `this.interpreter.arithmeticHelper.multiply(a, b)` helper?

Something we do for `(a: number, b: number) => this.interpreter.arithmeticHelper.subtract(a, b)`. 

The reasoning is to have a single place for computation, both are basicaly the same. Linking them with a shared helper would make developers aware of that in the future.